docs/detections/prebuilt-rules/downloadable-packages/0-16-1/prebuilt-rule-0-16-1-azure-ad-global-administrator-role-assigned.asciidoc

```diff
@@ -0,0 +1,79 @@
+[[prebuilt-rule-0-16-1-azure-ad-global-administrator-role-assigned]]
+=== Azure AD Global Administrator Role Assigned
+
+In Azure Active Directory (Azure AD), permissions to manage resources are assigned using Roles. The Global Administrator is a role that enables users to have access to all administrative features in Azure AD and services that use Azure Active Directory identities like the Microsoft 365 Defender portal, the Microsoft 365 compliance center, Exchange, SharePoint Online, and Skype for Business Online. Attackers can add users as Global Administrators to maintain access and manage all subscriptions and their settings and resources.
```

Comment:
Recommend lowercasing "roles" in the first sentence and changing "Azure Active Directory" to "Azure AD" in the second.